docs/development.rst

```diff
@@ -204,6 +204,11 @@ Swift conventions.
     library. The tests are written using the
     [Googletest](https://code.google.com/p/googletest/) framework.
 
+  **SwiftPM Unit Tests**
+
+    These tests are located under `unittests/Swift`. The tests are written using
+    the XCTest framework. Run them by executing `swift test` in the root level.
+
```

Comment:
The line after this is false, and I will want to fix it. By doing so, we will then immediately get these tests running in CI. The set up is complex though so I propose we merge this and I can take a look at the lot integration.

We actually already have a bit of it with the perftests (which are XCTest), so what is like to do is rename perftests to XCTests, sink perftests down one level, and move these into an adjacent directory.

Reply:
But the perf tests are probably running through `xcodebuild` and not via `swift test`? Would be great to have it integrated in the lit tests! 👍